openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -160,7 +160,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(37, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
+        with self.assertNumQueries(38, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
```

Comment:
Do you happen to know what added a query?  Is this the messages?  Just curious, no big deal.